crates/miden-lib/src/transaction/events.rs

```diff
@@ -53,6 +53,9 @@ const LINK_MAP_GET_EVENT: u32 = 0x2_001b; // 131099
 
 const UNAUTHORIZED_EVENT: u32 = 0x2_001c; // 131100
 
+// TODO: Move between epilogue start/end.
+const EPILOGUE_AFTER_COMPUTE_FEE_PROCEDURE: u32 = 0x2001d; // 131101
```

Comment:
Would addressing this `TODO` in this PR complicate something?

Also, nit: should we call this event `EPILOGUE_AFTER_TX_FEE_COMPUTED`?